app/build.gradle.kts

```diff
@@ -56,6 +59,9 @@ android {
 
         manifestPlaceholders["MAPS_API_KEY"] = mapsApiKey
         buildConfigField("String", "PERSPECTIVE_API_KEY", "\"${perspectiveApiKey}\"")
+        buildConfigField("String", "DIGITAL_OCEAN_SPACE_ACCESS_KEY", "\"${digitalOceanSpaceAccessKey}\"")
+        buildConfigField("String", "DIGITAL_OCEAN_SPACE_SECRET_KEY", "\"${digitalOceanSpaceSecretKey}\"")
+
```

Comment:
Question: Did you check that the builder APK takes the correct secrets?

Reply:
nope, was thinking of doing this in the upcoming pr. Would be easier to test as the pictures would be implemented there.